apps/books/templates/categoryvisebooks.html

```diff
@@ -0,0 +1,28 @@
+<h1>{{ object.title }}:</h1>
```

Comment:
How about naming the file `category_books.html`?

Reply:
Great